packages/suite-native-settings/src/components/ApplicationSettings.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { ListItem } from '@suite-native/atoms';
+
+import { SettingsSection } from './SettingsSection';
+
+import { SettingItem } from '../types';
+import { SettingsStackRoutes } from '../navigation/routes';
+
+type ApplicationSettingsProps = {
+    onRedirect: (route: SettingsStackRoutes | undefined) => void;
+};
+
+const applicationSettingsItems: SettingItem[] = [
+    {
+        title: 'Localisation',
+        description: 'Language, Currency',
+        iconName: 'flag',
+        route: SettingsStackRoutes.SettingsLocalisation,
+    },
+    {
+        title: 'Labeling',
+        description: 'Saved locally',
+        iconName: 'label',
+        route: SettingsStackRoutes.SettingsLabeling,
+    },
+    {
+        title: 'Advanced',
+        description: 'Usage data, Logs',
+        iconName: 'eyeglasses',
+        route: SettingsStackRoutes.SettingsAdvanced,
+    },
+];
+
+export const ApplicationSettings = ({ onRedirect }: ApplicationSettingsProps) => (
+    <SettingsSection title="Aplication">
+        {applicationSettingsItems.map(item => (
+            <ListItem
+                key={item.title}
+                iconName={item.iconName}
```

Comment:
why not spread this attrs?

Reply:
I don't know. Is it important? I usually don't give it a second thought in map functions to spread if I'm not using properties more than once. Are you up for spreading them? If so, we should always do this in other components as well.

Reply:
[e035a1b](https://github.com/trezor/trezor-suite/pull/5617/commits/e035a1b7e5eabc6a3f1ee38466cc48311713970f)